Migrate text from guideline (#115)
clearhaus/gateway-api-docs: source/index.html.md

API keys can have an expire date. API keys can be enabled or disabled. For
merchant accounts, these details can be observed in the Clearhaus Dashboard.

> For merchant accounts, the use of one active (enabled and unexpired) API key is
> indistinguishable from the use of another active API key. An authorization made
> with one API key can be captured using any active API key for that same merchant
> account.
> This is in contrast to accounts which are independent; an authorization made on
> one merchant account cannot be captured on another account.

Reviewer: ```suggestion
For a merchant account, the use of one active (enabled and unexpired) API key
is indistinguishable from the use of another active API key. An authorization
made with one API key can be captured using any active API key for that same
merchant account. In contrast, an authorization made on one merchant account
cannot be captured on another account.
```

Author: Just wrapping, right?

API keys can have an expire date. API keys can be enabled or disabled. For
merchant accounts, these details can be observed in the Clearhaus Dashboard.

> An API key points to an account. An account can have multiple API keys. For

Reviewer: Very similar to line 89.

Author: Resolved by 9e24cc12c69ca2c7a5118266cbd5c8512baa1fe4
